docs2/site/docs/migrations/migration6.md

```diff
@@ -54,6 +54,10 @@ Field<IntGraphType>("id").WithComplexityImpact(123);
 
 For more details, please review the PR here: https://github.com/graphql-dotnet/graphql-dotnet/pull/3159
 
+### 6. `AutoRegisteringObjectGraphType` recognizes inherited methods
+
+Inherited methods are now recognized by `AutoRegisteringObjectGraphType` and fields are built for them.
```

Comment:
🤔 Does it make sense to do the same for inputs?

Reply:
Methods cannot be written to, and so are ignored by the auto input type. Inherited properties already work for both input and output types.

Reply:
ok